person.rb

```diff
@@ -18,4 +19,15 @@ def can_use_services?
   def of_age?
     @age >= 18
   end
+
+  def correct_name
+    @name
+  end
```

Comment:
- [ ] I will encourage you to make this method public so that methods from other files can access it. At the moment, I have attached the error I get to this comment. The first error is the error message you're seeing in person.rb:31 indicates that you are trying to call a private method called correct_name on an instance of the Person class. However, the correct_name method is not accessible outside the Person class because it is private. I implore you to make that very method public.
- [ ] The error message you're seeing in person.rb:33 indicates that you are trying to create a new instance of a class called CapitalizeDecorator, but Ruby can't find the definition of this class. The NameError is telling you that the constant CapitalizeDecorator is uninitialized, meaning it has not been defined or loaded into the current context. I would advise you to require_relative './decorator' so that it can be accessed.


![image](https://user-images.githubusercontent.com/86200356/232853639-f7b3e699-90d7-4970-8c31-257e1d4d8ba3.png)